.github/workflows/build-validation.yml

```diff
@@ -1,28 +1,12 @@
 # This is a basic workflow to help you get started with Actions
-name: Snippets 5000
+name: 'Snippets 5000'
 
 # Controls when the action will run. Triggers the workflow on push or pull request
 # events on the main branch only.
 on:
-  pull_request:
-    paths:
-      - "**.cs"
-      - "**.vb"
-      - "**.fs"
-      - "**.cpp"
-      - "**.h"
-      - "**.xaml"
-      - "**.razor"
-      - "**.cshtml"
-      - "**.vbhtml"
-      - "**.csproj"
-      - "**.vbproj"
-      - "**.fsproj"
-      - "**.vcxproj"
-      - "**.sln"
-      - "**global.json"
-      - "**snippets.5000.json"
+  pull_request_target:
```

Comment:
Do we want to run this for .md only file changes? I think we still might want the `paths` filter, if it is applicable.

Reply:
That's the point: If this action doesn't run, it gets marked as "pending", and the PR is stuck. I did run some tests, and this won't use many resources when there aren't any code changes. The LocateProjects exe will just look at the PR paths and ignore them. One exception might be CI updates in the API repo, where thousands of files change. The link above in the description explains the design from GitHub's support team.